### ISSUE TITLE
Error: 429 - Too Many Requests

### ISSUE THREAD

**Issue by MustachedManiac** [ytmdesktop/ytmdesktop#1240]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

I'm currently receiving the error 429, too many quests when attempting to GET /STATE.  I understand what the error means, but what I can't find is what the actual limits are?  100/day? 10/minute?  Any guidance would be greatly appreciated.

### Expected Behavior

I expected to know how many times I can ping the endpoint before getting hit with rate limits.

### Steps To Reproduce

Set up an action that pings the api end point.  Mine was set to hit it every 10 seconds.  

### YTMDesktop

2.0.0-rc.5

### OS

Windows

### OS Version

11

### Arch

x64

### Installation way

.exe

### Anything else?

Would be really awesome to incorporate the ability for just a basic websocket that I could listen to, instead of Socket.IO.  Unfortunately I'm unable to connect via Socket.IO when using C# due to the inability for me to send a proper config file?  Unless you know a workaround, a basic websocket that I could just listen to without the need for Socket.IO would make my use case infinitely better

**Comment by Alipoodle** [comment]
https://github.com/ytmdesktop/ytmdesktop/blob/50d2a6e452961905289bf6eed2f2d45efed0d834/src/main/integrations/companion-server/api/v1/index.ts#L452
1 Every 5 seconds.

Comment on it says
>  API users: Please utilize the realtime websocket to get the state. Request this endpoint as necessary, such as initial state fetching.

**Comment by MustachedManiac** [comment]
> https://github.com/ytmdesktop/ytmdesktop/blob/50d2a6e452961905289bf6eed2f2d45efed0d834/src/main/integrations/companion-server/api/v1/index.ts#L444-L453
> 
> 
> 1 Every 5 seconds.
> Comment on it says
> 
> > API users: Please utilize the realtime websocket to get the state. Request this endpoint as necessary, such as initial state fetching.

I didn't even think to reference the V1 docs, thank you!  That being said.  Not sure why it gave 429 if I was pinging every 10 seconds then?

**Comment by NovusTheory** [comment]
Seems like this issue is resolved. I'm going to close this then

> Still hoping it's possible to have a WS alternative without Socket.IO

I initially thought of doing a raw websocket when I made v2 but I opted to stick with socket.io for convenience reasons. Most languages have a socket.io library but if yours doesn't then you unfortunately have to write a library for it.

> I expected to know how many times I can ping the endpoint before getting hit with rate limits.

For future reference read the headers of the request
